docs/modules/ROOT/pages/accept-invitation.adoc

```diff
@@ -0,0 +1,22 @@
+= Accept Invitation to Organization
+:description: When the admin user invites someone to join the organization, an invitation is sent to the specified email address. 
+
+{description}
+
+To accept an invitation to join an organization, complete the following steps:
+
+. Open the email from your organization
+. Select the *ACCEPT INVITE* button
```

Comment:
Click is not a term used in tech docs since multiple interfaces came into use - you cannot click a touch-screen, for instance. Select has been the standard term since this time. 

However, I understand that select can mean highlight, for example to copy the selected text.

I'll discuss with the wider doc team and we'll start to standardise the terms based on the decision.

Reply:
Had a discussion with the wider docs team; we are going to go with the MS Style Guide, which uses Select. 

See https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui?source-recommendations if you want to read more ;-)